src/tests/ov_helpers/ov_snippets_models/src/subgraph_mlp.cpp

```diff
@@ -0,0 +1,260 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "subgraph_mlp.hpp"
+
+#include "snippets/op/subgraph.hpp"
+
+#include "openvino/opsets/opset15.hpp"
+#include "ov_lpt_models/common/builders.hpp"
+#include "ov_lpt_models/common/dequantization_operations.hpp"
+#include "ov_lpt_models/common/fake_quantize_on_data.hpp"
+
+namespace ov {
+namespace test {
+namespace snippets {
+
+std::shared_ptr<ov::Model> MLPSeqFunction::initOriginal() const {
+    auto A_param = std::make_shared<ov::op::v0::Parameter>(precisions[0], input_shapes[0]);
+    std::shared_ptr<Node> current = A_param;
+
+    auto mlp_layer = [&](size_t m, size_t n) {
+        auto b_shape = ov::Shape{m, n};
+        auto b_row = ov::Shape{m};
+        auto B = std::make_shared<ov::op::v0::Constant>(ov::element::f32, b_shape, std::vector<float>{0.1122f});
+        current = std::make_shared<ov::op::v0::MatMul>(current, B, false, true);
+        auto constant = std::make_shared<ov::op::v0::Constant>(ov::element::f32,
+            b_row,
+            std::vector<float>{0.1122f});
```

Comment:
Could you please change constant values on each `mlp_layer` call? We need it to properly cover binary postops fusion pass. The same for `MLPSeqQuantizedFunction`

Reply:
Done